project-base/app/tests/FrontendApiBundle/Functional/Cart/ApplyPromoCodeToCartTest.php

```diff
@@ -68,15 +68,17 @@ public function testApplyPromoCode(): void
                 promoCode: "' . $promoCode->getCode() . '"
             }) {
                 uuid
-                promoCode
+                promoCodes {
+                    code
```

Comment:
could you please add all the fields (code, type, discount) to the assertions in all the affected tests?

Reply:
I have added it only to this one and created second one for nominal promo code. I do not feel, that including it in all tests brings any benefit.